meson.build

```diff
@@ -60,16 +60,25 @@ if host_machine.system() != 'windows'
 endif
 thread_dep = dependency('threads')
 
-libdrm_dep = dependency('libdrm', required : get_option('encoder'))
-libvhal_dep = dependency('vhal-client',
-  required : get_option('encoder').enabled() or get_option('streamer').enabled())
-libvpl_dep = dependency('vpl', required : get_option('encoder'))
-libva_dep = dependency('libva', required : get_option('encoder'))
+if host_machine.system() == 'linux'
+  libdrm_dep = dependency('libdrm', required : get_option('encoder'))
+  libvhal_dep = dependency('vhal-client',
+    required : get_option('encoder').enabled() or get_option('streamer').enabled())
+  libvpl_dep = dependency('vpl', required : get_option('encoder'))
+  libva_dep = dependency('libva', required : get_option('encoder'))
+
+  libavcodec_dep = dependency('libavcodec', required : get_option('encoder'))
+  libavfilter_dep = dependency('libavfilter', required : get_option('encoder'))
+  libavformat_dep = dependency('libavformat', required : get_option('encoder'))
+  libavutil_dep = dependency('libavutil', required : get_option('encoder'))
+
+  titansdk_dep = null_dep
+endif
 
-libavcodec_dep = dependency('libavcodec', required : get_option('encoder'))
-libavfilter_dep = dependency('libavfilter', required : get_option('encoder'))
-libavformat_dep = dependency('libavformat', required : get_option('encoder'))
-libavutil_dep = dependency('libavutil', required : get_option('encoder'))
+if host_machine.system() == 'windows'
+  owt_dep = null_dep
+  titansdk_dep = null_dep
```

Comment:
Same as the linux def, may be move outside the if/else blocks?

Reply:
That's a placeholder for windows nulling dependencies. `titansdk_dep` will actually be completely removed even for windows, but this will make more difficult for me to cherry-pick changes from internal codebase if I will start to change this right now. So, looking for middle ground I left it here for now.

Reply:
Sounds good. titansdk_dep is pruned in the next commit. This particular reference is retained to facilitate code merges from internal branches.